configs/roles/subscriptions.json

```diff
@@ -8,7 +8,8 @@
       "version": 2,
       "access": [
         {
-          "permission": "subscriptions:*:*"
+          "permission": "subscriptions:*:*",
+          "permission": "inventory:*:read"
```

Comment:
@iphands just double checking on this - thought we chatted about adding read-only inventory permissions to all insights apps except for subs watch?

Reply:
I'll check with them on this